fecha.js

```diff
@@ -237,9 +238,21 @@
 
     mask = fecha.masks[mask] || mask || fecha.masks['default'];
 
-    return mask.replace(token, function ($0) {
+    var literals = [];
+
+    //Make literals inactive by replacing them with _
+    mask = mask.replace(literal, function($0, $1) {
+      literals.push($1);
+      return '_';
```

Comment:
Can we use something more unlikely to be used? Someone may put a `_` and break this. How about `??`